include/tools.h

```diff
@@ -216,13 +217,13 @@ std::map<std::string, std::string> getNetworkInterfaces();
 /** Provides the best IP address
  * This function provides the best IP address from the list given by getNetworkInterfacesIPv4Or6()
  */
-std::string getBestPublicIp(bool ipv6);
+std::vector<std::string> getBestPublicIps(IpMode mode);
```

Comment:
What about changing the return type to `IpAddress` (and reverting the function name change)?

Reply:
Thanks for this.

Reply:
@veloman-yunkan what would you say about making it `IpMode getBestPublicIps(void)`?

Reply:
I don't get your proposal. Why would a function named `getBestPublicIps()` return an `IpMode` instead of an IP address?

Reply:
Excuse my typo I meant `IpAddress getBestPublicIps(void)`

Reply:
Makes sense.